apis/workload/v1alpha1/types.go

```diff
@@ -79,7 +82,7 @@ type KubernetesApplicationResourceTemplate struct {
 // KubernetesApplicationStatus represents the status of a Kubernetes
 // application.
 type KubernetesApplicationStatus struct {
-	corev1alpha1.ConditionedStatus
+	corev1alpha1.ConditionedStatus `json:"conditionedStatus,omitempty"`
```

Comment:
is tagging the types a new requirement in V2, or did we just miss it?

Reply:
Tagging is required by `controller-tools` in V2.